session-6/task4/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+
+func main() {
+	countries := map[string]string{
+		"Azerbaijan": "Baku",
+		"USA":        "Washington",
+		"Germany":    "Berlin",
+		"Japan":      "Tokyo",
+	}
+	countries["Turkey"] = "Ankara"
+	countries["Italy"] = "Rome"
+	for k := range countries {
+		fmt.Printf("Capital of %s is %s\n", k, getCountryName(k, countries))
+	}
+	fmt.Println(getCountryName("France", countries))
+}
+
+func getCountryName(s string, countries map[string]string) string {
+	var Red = "\033[31m"
+	var Reset = "\033[0m"
```

Comment:
Console coloring to Red is good point. So, why we need Reset?

Reply:
Without Reset, all the following text will be red.